src/apps/halcsd/init/systemd/etc/udev/rules.d/80-udev_fpga.rules

```diff
@@ -2,4 +2,4 @@
 
 KERNEL=="fpga*", ACTION=="add", OWNER="root", GROUP="users", MODE="660", PROGRAM="/usr/local/share/halcs/scripts/generate-dev-idx.sh %k", SYMLINK+="fpga/%c{1}", RUN+="/usr/bin/systemctl --no-block start halcs-ioc@%c{2}.target halcs-ioc@%c{3}.target"
```

Comment:
I'd first make a commit removing this file, it's not used by the build system anymore.

Reply:
I've fixed and properly parameterized the build system (`80-udev_fpga.rules` was actually being used by it) so it's not installed anymore.